suites/pacific/rbd/tier-1_rbd_mirroring_upgrade-4-to-latest.yaml

```diff
@@ -0,0 +1,223 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_rbd_mirror.yaml
+# Test-Case: Configure RBD Mirror setup and run IOs
+# Polarion ID : CEPH-83573329 - RBD HA MirroringDraft
+#
+# Cluster Configuration:
+#    cephci/conf/nautilus/rbd/tier-1_rbd_mirror.yaml
+#    No of Clusters : 2
+#    Each cluster configuration
+#    6-Node cluster(RHEL-8.3 and above)
+#    1 MONS, 1 MGR, 3 OSD and 1 RBD MIRROR service daemon(s)
```

Comment:
Can we please use 3 mon cluster for upgrade scenarios ?

Reply:
Done.